android/source/VideoLocker/src/org/edx/mobile/logger/OEXLogger.java

```diff
@@ -0,0 +1,31 @@
+package org.edx.mobile.logger;
+
+/**
+ * Created by shahid on 22/1/15.
+ */
+public class OEXLogger {
```

Comment:
You don't need to call things ``OEX`` on android, because Java has a built in namespace mechanism. That's only necessary in Objective-C where there's a single flat namespace.